### PR TITLE
Add a first README with install instructions, a first script to test …

### PR DESCRIPTION
**Pull request by jonathanrocher**
…installation and a gitignore.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+This repository contains all the material needed by students registered to the
+Numpy tutorial of `SciPy 2015<http://scipy2015.scipy.org/ehome/115969/289057/?&>`_
+on July Mon July 6th 2015.
+
+
+Packages needed
+---------------
+
+If you don't already have a working distribution, by far the easiest way to get
+everything you need for this turtorial is to download
```
Comment (kjordahl) [inline README.rst]: "tutorial"

File: test_python_install.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+""" Run this file with nosetests to check your python installation.
```
Comment (kjordahl) [inline test_python_install.py]: For consistency, this file should be called `check_env.py` and be runnable as a script. Adding an `if __name__ == "__main__"` clause that calls `nose.run()` should do the trick.

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
+pandas_tutorial
+===============
+
+This repository contains all the material needed by students registered to the
```
Comment (kjordahl) [inline README.rst]: "registered for the"

File: README.rst (hunk 1 above)
Comment (jonathanrocher) [inline README.rst]: Fixed.

File: test_python_install.py (hunk 2 above)
Comment (jonathanrocher) [inline test_python_install.py]: That doesn't work for me. Does that works for you?

File: README.rst (hunk 3 above)
Comment (jonathanrocher) [inline README.rst]: Fixed.

File: test_python_install.py (hunk 2 above)
Comment (kjordahl) [inline test_python_install.py]: It needs an argument: `nose.run(defaultTest=__name__)`

**Comment by kjordahl** [comment]
:+1: 
Thanks!

**Pull request merged**